packages/baml/baml_client/globals.ts

```
@@ -0,0 +1,67 @@
+
+export function resetBamlEnvVars(envVars: Record<string, string | undefined>) {
+  if (DO_NOT_USE_DIRECTLY_UNLESS_YOU_KNOW_WHAT_YOURE_DOING_CTX.allowResets()) {
+    const envVarsToReset = Object.fromEntries(Object.entries(envVars).filter((kv): kv is [string, string] => kv[1] !== undefined));
+    DO_NOT_USE_DIRECTLY_UNLESS_YOU_KNOW_WHAT_YOURE_DOING_RUNTIME.reset('baml_src', getBamlFiles(), envVarsToReset)
+    DO_NOT_USE_DIRECTLY_UNLESS_YOU_KNOW_WHAT_YOURE_DOING_CTX.reset()
+  } else {
+    throw new Error('BamlError: Cannot reset BAML environment variables while there are active BAML contexts.')
+  }
+}
+
+const patchedLoad = (originalFn: any) => (...args: any[]) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)